docs/rackhd/configuration.rst

```diff
@@ -209,6 +209,8 @@ The following table describes the configuration parameters in config.json:
       - Enable trust proxy in express. Populate req.ip with left most IP address from the XForwardFor list.
 
         See documentation at https://expressjs.com/en/guide/behind-proxies.html
+    * - obmOption
+      - Allow rackHD to setup IPMI OBM settings on active dicovery by creating a new BMC user on the compute node.
```

Comment:
Please add some more words to describe the type of obmOption, as due to our implementation in condition job, the type of obmOption is very tricky, it should be `a boolean string`.